Events/MaterialTradedEvent.cs

```diff
@@ -0,0 +1,60 @@
+﻿using EddiDataDefinitions;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace EddiEvents
+{
+    public class MaterialTradedEvent : Event
+    {
+        public const string NAME = "Material traded";
+        public const string DESCRIPTION = "Triggered when you trade materials at a materials trader";
+        public const string SAMPLE = "{ \"timestamp\": \"2018-04-02T05:04:45Z\", \"event\": \"MaterialTrade\", \"MarketID\": 3223343616, \"TraderType\": \"encoded\", \"Paid\": { \"Material\": \"shielddensityreports\", \"Material_Localised\": \"Untypical Shield Scans \", \"Category\": \"$MICRORESOURCE_CATEGORY_Encoded;\", \"Category_Localised\": \"Encoded\", \"Quantity\": 72 }, \"Received\": { \"Material\": \"shieldfrequencydata\", \"Material_Localised\": \"Peculiar Shield Frequency Data\", \"Category\": \"$MICRORESOURCE_CATEGORY_Encoded;\", \"Category_Localised\": \"Encoded\", \"Quantity\": 2 } }";
+        public static Dictionary<string, string> VARIABLES = new Dictionary<string, string>();
+
+        static MaterialTradedEvent()
+        {
+            VARIABLES.Add("tradertype", "The type of material trader for the trade");
+            VARIABLES.Add("paid", "The name of the material lost in the trade");
+            VARIABLES.Add("paidqty", "The amount of the material lost in the trade");
```

Comment:
Not keen on this abbreviation, it's too opaque for non-native speakers, would prefer "paid_quantity"